### PR TITLE
Adds octocat image instead of text link

### PR DESCRIPTION
**Pull request by rcorral**
**Option 1**
iimg

**Option 2**
m

**And it moves**
img

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by jfurrow** [comment]
Love it! I think it looks better at the bottom.

**Comment by rcorral** [comment]
Me too and that's how it is now.

**Comment by daiyiwhale** [comment]
daw

File: docs/styles/page-header.less [hunk 1]
```diff
@@ -125,6 +125,29 @@ Page Header
...
+    transform: rotate(-25deg);
+  }
+  40%, 80%{
+    transform: rotate(10deg);
+  }
+}
+
+@media (max-width:500px) {
+  .github-corner:hover .octo-arm {
+    animation: qnone;
```
Comment (jfurrow) [inline docs/styles/page-header.less]: I think you meant `none` here

File: docs/styles/page-header.less [hunk 2]
```diff
@@ -125,6 +125,29 @@ Page Header
...
+  }
+
+  .github-corner .octo-arm {
+    animation: octocat-wave 560ms ease-in-out
+  }
```
Comment (jfurrow) [inline docs/styles/page-header.less]: #nit Could we rewrite this block like so?

```css
.github-corner {

  &:hover {

    .octo-arm {
      animation: none;
    }
  }

  .octo-arm {
    animation: octocat-wave 0.5s ease-in-out;
  }
}
```

File: docs/styles/page-header.less [hunk 3]
```diff
@@ -125,6 +125,29 @@ Page Header
...
+  20%, 60% {
+    transform: rotate(-25deg);
+  }
+  40%, 80%{
+    transform: rotate(10deg);
+  }
+}
+
+@media (max-width:500px) {
```
Comment (jfurrow) [inline docs/styles/page-header.less]: #nit Can we put this media query at the end of the file?

**Comment by jfurrow** [comment]
Sorry for all the #nits, but this is adorable!

File: docs/styles/page-header.less (hunk 1 above)
Comment (jfurrow) [inline docs/styles/page-header.less]: Done!

File: docs/styles/page-header.less (hunk 2 above)
Comment (jfurrow) [inline docs/styles/page-header.less]: Done!

**Comment by jfurrow** [comment]
@rcorral I pushed some commits, feel free to merge if you're cool with 'em!

**Pull request merged**